aws/main.tf

```diff
@@ -150,8 +172,8 @@ resource "aws_instance" "server" {
     cloud_env                 = "aws"
     retry_join                = var.retry_join
     nomad_binary              = var.nomad_binary
-    nomad_consul_token_id     = var.nomad_consul_token_id
-    nomad_consul_token_secret = var.nomad_consul_token_secret
+    nomad_consul_token_id     = random_uuid.nomad_id.result
+    nomad_consul_token_secret = random_uuid.nomad_token.result
```

Comment:
It would be better to name `random_uuid.nomad_token.result` as `random_uuid.nomad_secret.result` OR name `nomad_consul_token_secret` as `nomad_consul_token_token`. Whichever works better.